src/base-element.js

```diff
@@ -503,8 +530,8 @@ export class BaseElement {
    * @param {!Element|!Array<!Element>} elements
    * @protected
    */
-  schedulePause(elements) {
-    this.resources_.schedulePause(this.element, elements);
+  scheduleUnload(elements) {
```

Comment:
I'm not sure yet. There a few elements which implement an `unlayoutCallback` (`amp-anim`, `amp-iframe`, etc)  and not a `pauseCallback`. In the cases where we were calling `schedulePause`, we definitely meant `scheduleUnload`.